layouts/partials/scripts.html

```diff
@@ -12,13 +12,14 @@
 <script src="{{ $js.RelPermalink }}" integrity="{{ $js.Data.Integrity }}" crossorigin="anonymous"></script>
 {{ end }}
 <script>
-  docsearch({
-    apiKey: '8904bbd3ca621bef472e3de7e0e29532',
-    indexName: 'jenkins_x',
-    inputSelector: '#search-input',
-    algoliaOptions: { 'facetFilters': ["lang:{{ .Page.Language.Lang }}"] },
-    debug: false
-  });
+
+docsearch({
+  container: '#docsearch',
+  appId: 'GIN1OS4UKS',
+  indexName: 'jenkins-x',
+  apiKey: 'a9c99be0d65d1a8f6059be9104a12cd4',
```

Comment:
Is this a search only api-key? https://www.algolia.com/doc/guides/security/api-keys/#search-only-api-key

Reply:
Yes it is